src/containers/Create/index.jsx

```diff
@@ -70,20 +71,23 @@ export default class Create extends React.Component {
           button_text={this.state.button_text}
           onClick={() => this._create()}
         />
+        <text style={Styles.TextContractAddress}>
+          {this.state.contract_address}
+        </text>
       </div>
     );
   }
 
   _create() {
     if (
-      this.state.input_totally_supply !== "" &&
-      this.state.input_name !== "" &&
-      this.state.input_decimal_places !== "" &&
-      this.state.input_symbol !== ""
+      this.state.input_totally_supply !== '' &&
```

Comment:
just use `!this.state.input_totally_supply` instead of `this.state.input_totally_supply !== ''`